docs/stencil-docs/installing-stencil-cli/live-previewing-a-theme.md

```diff
@@ -114,6 +114,19 @@ The Browsersync preview is launched by executing the `stencil start` command in
 * an API access token
 * a local port number
 
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme: info -->
+
+### Note  
+The store's URL should be a vanity URL.

```

Comment:
```suggestion
The store's URL should be a vanity URL which is a bla bla bla bla bla
```
**demo comment** - you can delete this, but if you look below the comment there should now be a button that will automatically make my suggested changes